lightning/src/util/events.rs

```diff
@@ -174,7 +177,7 @@ impl core::fmt::Display for ClosureReason {
 		match self {
 			ClosureReason::CounterpartyForceClosed { peer_msg } => {
 				f.write_str("counterparty force-closed with message ")?;
-				f.write_str(&peer_msg)
+				f.write_str(&format!("{}", peer_msg))
```

Comment:
I _think_ you can also do the following, which would avoid an intermediate string allocation.

```rust
f.write_fmt(format_args!("counterparty force-closed with message: {}", peer_msg))
```

Reply:
ah cool, ya thats a bit better